### ISSUE TITLE
Channel stuck opening - Pending

### ISSUE THREAD

**Issue by indomitorum** [lightningnetwork/lnd#6048]
### Background

I opened a channel today but even tho it shows as pending on TH, RTL and Umbrel dashboard, it doesn't show up in the mempool. 

It just says transaction not found.

### Your environment

* version of `lnd` - 14.1
* which operating system (`uname -a` on *Nix) - Debian - Raspberry Pi 4 - 8Gb. Umbrel 0.4.9
* version of `btcd`, `bitcoind`, or other backend - Bitcoin Core 22.0

### Steps to reproduce

I opened a channel today but even tho it shows as pending on TH for example, it doesn't show up in the mempool. 

It just says transaction not found. 

https://mempool.space/tx/3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563

lncli pending channels shows this:

lncli pendingchannels
{
    "total_limbo_balance": "2367232",
    "pending_open_channels": [
        {
            "channel": {
                "remote_node_pub": "02fe80fb6a2dc0fb6e9bec49c76d048889c91355d4e900fcb026bf095665790325",
                "channel_point": "3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1",
                "capacity": "4500000",
                "local_balance": "4496530",
                "remote_balance": "0",
                "local_chan_reserve_sat": "45000",
                "remote_chan_reserve_sat": "45000",
                "initiator": "INITIATOR_LOCAL",
                "commitment_type": "ANCHORS",
                "num_forwarding_packages": "0"
            },
            "confirmation_height": 0,
            "commit_fee": "2810",
            "commit_weight": "772",
            "fee_per_kw": "2500"

Lnd logs show this :

2021-12-02 06:54:46.588 [INF] FNDG: Generated ChannelPoint(3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1) for pending_id(2870300d2b72be9665f7976f757b11a3e4565d02c7b30ee1c7defb1cc675520f)
2021-12-02 06:54:46.936 [INF] FNDG: Broadcasting funding tx for ChannelPoint(3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1): 02000000000103128f9da1258b8775f1e528407024c229010f0a8b0523f4fb8ea86294dd3eb76a00000000000000000092acf08ba4f60e51c1dd071afa2fd476fbfdca2894033c611e56531e39b34871000000000000000000848f94c56a63c856c6fd708f83d943cbca8739467e8001e540d6b5910fe618de0000000000000000000280310e0000000000160014456cca7b81277a871927c6cc5a05df4ae06f68a420aa4400000000002200203fd82c2c21c989d9d458b181952d3993919624df33e1fb417a94069c493a3ddc02483045022100d899c2c25c6dd1a247cb9f751338c7c5a1f38682e232b5e99028ae96cb340c0d022070348b6c9a2de9443e5efd61d7d85d707e61d8b0b6588d29e25564fc4a9e844601210380149e125cf2d81ef2e9002efe459322ddb868085865443ac6267efaa692871e02483045022100ab4d67fbbf37e1630beccf4cd4d1868d57cb6a18b57e09cbfc3fad54dc90bef302205c3619f88a232fbeb5d866f37ca559cab7603cbceca239ff81f1e5949a32594d012103b22dbe4acb219fb4cbe627b06698ad5fcbba1712953dbdc9d0fa4ae411747c7002473044022004741012b2c2d2462c7869502a87ba5dcfd0809c5c995440ed24958d0802309f0220420a6c24ead32aaaf387d59b993c1af473c4617137b92ca06f18aaa0f9fff504012102f888f116689c7a1905d5ef53be8d6b9e131a1d7fa72c18e8bec0b2f29ce9926c00000000
2021-12-02 06:54:46.936 [INF] LNWL: Inserting unconfirmed transaction 3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563
2021-12-02 06:54:46.937 [ERR] LNWL: Notifying unmined tx notification (3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563) while creating notification for blocks
2021-12-02 06:54:46.962 [ERR] FNDG: Unable to broadcast funding tx 02000000000103128f9da1258b8775f1e528407024c229010f0a8b0523f4fb8ea86294dd3eb76a00000000000000000092acf08ba4f60e51c1dd071afa2fd476fbfdca2894033c611e56531e39b34871000000000000000000848f94c56a63c856c6fd708f83d943cbca8739467e8001e540d6b5910fe618de0000000000000000000280310e0000000000160014456cca7b81277a871927c6cc5a05df4ae06f68a420aa4400000000002200203fd82c2c21c989d9d458b181952d3993919624df33e1fb417a94069c493a3ddc02483045022100d899c2c25c6dd1a247cb9f751338c7c5a1f38682e232b5e99028ae96cb340c0d022070348b6c9a2de9443e5efd61d7d85d707e61d8b0b6588d29e25564fc4a9e844601210380149e125cf2d81ef2e9002efe459322ddb868085865443ac6267efaa692871e02483045022100ab4d67fbbf37e1630beccf4cd4d1868d57cb6a18b57e09cbfc3fad54dc90bef302205c3619f88a232fbeb5d866f37ca559cab7603cbceca239ff81f1e5949a32594d012103b22dbe4acb219fb4cbe627b06698ad5fcbba1712953dbdc9d0fa4ae411747c7002473044022004741012b2c2d2462c7869502a87ba5dcfd0809c5c995440ed24958d0802309f0220420a6c24ead32aaaf387d59b993c1af473c4617137b92ca06f18aaa0f9fff504012102f888f116689c7a1905d5ef53be8d6b9e131a1d7fa72c18e8bec0b2f29ce9926c00000000 for ChannelPoint(3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1): transaction rejected: output already spent
2021-12-02 06:54:46.962 [INF] CNCT: Creating new ChannelArbitrator for ChannelPoint(3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1)
2021-12-02 06:54:46.999 [INF] CNCT: ChannelArbitrator(3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1): starting state=StateDefault, trigger=chainTrigger, triggerHeight=712202
2021-12-02 06:54:47.000 [INF] NTFN: New spend subscription: spend_id=77, outpoint=3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1, script=0 3fd82c2c21c989d9d458b181952d3993919624df33e1fb417a94069c493a3ddc, height_hint=712202
2021-12-02 06:54:47.001 [INF] NTFN: Dispatching historical spend rescan for outpoint=3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1, script=0 3fd82c2c21c989d9d458b181952d3993919624df33e1fb417a94069c493a3ddc, start=712202, end=712202
2021-12-02 06:54:47.064 [INF] FNDG: Finalizing pending_id(2870300d2b72be9665f7976f757b11a3e4565d02c7b30ee1c7defb1cc675520f) over ChannelPoint(3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1), waiting for channel open on-chain

2021-12-02 06:54:47.066 [INF] NTFN: New confirmation subscription: conf_id=7, txid=3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563, num_confs=3 height_hint=712202
2021-12-02 06:54:47.067 [INF] CNCT: Close observer for ChannelPoint(3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1) active
2021-12-02 06:54:47.067 [INF] FNDG: Waiting for funding tx (3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563) to reach 3 confirmations
2021-12-02 06:54:47.116 [INF] NTFN: Historical spend dispatch finished for request outpoint=3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1, script=0 3fd82c2c21c989d9d458b181952d3993919624df33e1fb417a94069c493a3ddc (start=712202 end=712202) with details: <nil>
2021-12-02 08:46:32.642 [WRN] CHBU: Replacing disk backup for ChannelPoint(3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1) w/ newer version
2021-12-02 08:46:32.948 [WRN] CHBU: Replacing disk backup for ChannelPoint(3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1) w/ newer version

lncli wallet pendingsweeps
{
        "pending_sweeps": [
                {
                        "outpoint": "cda6066f69ba3cae2300c83b86b576a2ce1805e18a63348c94955662cea38655:1",
                        "witness_type": "COMMITMENT_ANCHOR",
                        "amount_sat": 330,
                        "sat_per_vbyte": 1,
                        "broadcast_attempts": 9,
                        "next_broadcast_height": 712312,
                        "requested_sat_per_vbyte": 1,
                        "requested_conf_target": 0,
                        "force": false
                }
        ]
}

### Expected behaviour

Channel open

### Actual behaviour

Channel does not open and tx does not show in the mempool

ThunderHub shows the funds as pending under lightning

A solution to this without losing the funds would be appreciated

**Comment by guggero** [comment]
Your problem is: `Unable to broadcast funding tx ... for ChannelPoint(3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563:1): transaction rejected: output already spent`

Something is wrong with your wallet state. Try restarting `lnd` with `reset-wallet-transactions=1` in your config file, that will re-scan your UTXOs. Once your funds are represented correctly again, you'll probably need to remove the pending channel with `lncli abandonchannel` since it will never open in the first place.

**Comment by indomitorum** [comment]
I see. So you suggest adding that line under applications options and restard lnd. 

What would be the exact syntax of lncli abandonchannel ?.  I have looked at the --help command but am unsure.

Would this be correct or do I need to add -i too?. Thats the TxID

lncli abandonchannel 3ff9d7d37fc9cb11a250069da2a30df7f3382bbd770601c95546edea66fad563

Many thanks

**Comment by indomitorum** [comment]
I have restarted lnd with reset-wallet-transactions in the config file. It seems to be taking ages to restart properly. Keep finding this in the lnd.log

2021-12-02 16:03:30.043 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:30.057 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:30.099 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:30.099 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:31.120 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:03:31.432 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:03:31.638 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:31.647 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:31.662 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:31.702 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:31.705 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:34.845 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:34.867 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:34.867 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:34.910 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:34.910 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:41.036 [INF] RPCS: New transaction subscription
2021-12-02 16:03:41.253 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:41.279 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:41.279 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:41.321 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:41.321 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:50.980 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:03:51.740 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:03:54.063 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:54.086 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:54.090 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:54.128 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:03:54.128 [ERR] RPCS: [/lnrpc.Lightning/ConnectPeer]: server is still in the process of starting
2021-12-02 16:04:07.629 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:04:07.635 [INF] RPCS: Canceling transaction subscription
2021-12-02 16:04:10.775 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:04:11.515 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:04:30.873 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:04:31.224 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:04:50.961 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:04:51.657 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:05:10.937 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator
2021-12-02 16:05:11.256 [ERR] RPCS: [/lnrpc.Lightning/PendingChannels]: unable to find arbitrator

Is this normal?

**Comment by indomitorum** [comment]
sorted. Took 5 hours to reset the wallet. To abandon I used docker exec lnclo abandonchannel --funding_txid --output --i_know_what_i_am_doing

**Comment by Roasbeef** [comment]
Closing as the base issue looks to be resolved, we have another issue tracking oddities with the wallet and certain unconfirmed transaction cases.

**Comment by zerofeerouting** [comment]
Just for anybody looking at this ticket - it seems to be correlated with this issue: https://github.com/lightningnetwork/lnd/issues/6241#issuecomment-1033738898
